src/components/display/fewlines/UpdateIdentityForm.tsx

```diff
@@ -2,19 +2,37 @@ import { useRouter } from "next/router";
 import React from "react";
 import styled from "styled-components";
 
-import { Identity, ReceivedIdentityTypes } from "../../../@types/Identity";
 import { Button, ButtonVariant } from "./Button/Button";
 import { Input } from "./Input/Input";
+import { NavigationBreadcrumbs } from "./NavigationBreadcrumbs/NavigationBreadcrumbs";
+import {
+  Identity,
+  IdentityTypes,
+  ReceivedIdentityTypes,
+} from "@src/@types/Identity";
+import { Box } from "@src/components/display/fewlines/Box/Box";
 
 export const UpdateIdentityForm: React.FC<{
   updateIdentity: (newValue: string) => Promise<void>;
   currentIdentity: Identity;
 }> = ({ currentIdentity, updateIdentity }) => {
   const [identity, setIdentity] = React.useState("");
   const router = useRouter();
+  const { value } = currentIdentity;
 
   return (
     <Wrapper>
+      <NavigationBreadcrumbs
+        title="Logins"
+        breadcrumbs={`${
+          currentIdentity.type === IdentityTypes.EMAIL.toLowerCase()
```

Comment:
We should really settle about using `IdentityTypes` vs `ReceivedIdentityTypes`.

I really don't like that we have 2 sources of verity for identity types. Also, the end goal will be to have `IdentityTypes` type coming from the SDK.

I'll let it like this for now, but when my PR is merged, we'll need to think about this.